<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventManagementWindow</class>
 <widget class="QMainWindow" name="EventManagementWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushFree2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Освободить кладовщика 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushGo">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить рабочего</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushFree5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>140</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Освободить кладовщика 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushFree">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Освободить кладовщика 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushFree3">
    <property name="geometry">
     <rect>
      <x>416</x>
      <y>230</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Освободить кладовщика 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushFree4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Освободить кладовщика 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
